util/config/config.go

```diff
@@ -431,3 +436,33 @@ func (config *Configuration) CheckPorts(myIP string) (bool, error) {
 	}
 	return true, nil
 }
+
+func OpenConfigFile() ([]byte, error) {
+	var file []byte
+
+	configFile := ConfigFile
+	if configFile == "" {
+		configFile = DefaultConfigFile
+	}
+	_, err := os.Stat(configFile)
+	if err != nil {
+		return file, err
```

Comment:
We should return `nil, error` here to make sure we don't accidentally return non-nil file if error is not nil